examples/pytorch/vicregl.py

```diff
@@ -36,20 +34,16 @@ def forward(self, x):
 device = "cuda" if torch.cuda.is_available() else "cpu"
 model.to(device)
 
-pascal_voc = torchvision.datasets.VOCDetection(
-    "datasets/pascal_voc", download=True, target_transform=lambda t: 0
-)
 transform = VICRegLTransform(n_local_views=0)
-dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=transform)
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
+)
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?